tests/sentry/tasks/test_check_auth.py

```diff
@@ -20,9 +21,12 @@ def test_simple(self, mock_check_auth_identity):
         auth_provider = AuthProvider.objects.create(
             organization_id=organization.id, provider="dummy"
         )
-        OrganizationMember.objects.create(
-            user_id=user.id, organization=organization, flags=OrganizationMember.flags["sso:linked"]
-        )
+        with assume_test_silo_mode(SiloMode.REGION):
+            OrganizationMember.objects.create(
```

Comment:
You can use factories instead of `assume_silo_mode`, e.g. https://github.com/getsentry/sentry/blob/c594324ef8acfed929271b4b2323694a2401c4b4/src/sentry/testutils/factories.py#L311